utils/problems.py

```diff
@@ -193,6 +194,10 @@ async def fetch_random_question(
         return
 
     try:
+        # If the question is premium only, fetch a different question.
+        if response_data["data"]["randomQuestion"]["isPaidOnly"]:
+            return await fetch_random_question(bot, difficulty)
+
```

Comment:
I don't personally agree with a recursive approach here, mainly due to scalability/resource-limitation concerns, resilience, and an unspecified maximum amount of tries. I recommend a `for` loop that will attempt getting an unpaid question a specific amount of times at max (let's say 10 for now - make sure to create a variable that makes it clear what the 10 represents - alongside a comment on why the for loop is necessary), that encompasses the whole of lines 190 up until basically the end of the function (so, including the try, except).

Although, these are very precautionary steps I'm suggesting, as it is very unlikely to require many attempts to finally get an unpaid question, it's still something I think is good to handle, just in case - who knows.